docs/source/topics/proc_accessing-the-internal-openshift-registry.adoc

```diff
@@ -0,0 +1,92 @@
+[id="accessing-the-internal-openshift-registry_{context}"]
+= Accessing the internal OpenShift registry
+
+The OpenShift cluster running in the {prod} virtual machine includes an internal container image registry by default.
+This internal container image registry can be used as a publication target for locally developed container images.
+To access the internal OpenShift registry, follow these steps.
+
+.Prerequisites
+
+* A running {prod} virtual machine.
+For more information, see link:{crc-gsg-url}#starting-the-virtual-machine_gsg[Starting the virtual machine].
+* A working [command]`oc` command.
+For more information, see link:{crc-gsg-url}#accessing-the-openshift-cluster-with-oc_gsg[Accessing the OpenShift cluster with `oc`].
+* For Docker, add `default-route-openshift-image-registry.apps-crc.testing` as an insecure registry.
```

Comment:
what about podman ? (which is used in this doc)

Reply:
There is no prerequisite for using `podman` beyond having `podman` installed. It's entirely possible to add the internal registry as an insecure registry for `podman`, but this isn't strictly necessary as long as you log in via `podman` with the  `--tls-verify=false` flag (as documented).

Perhaps we could add "an installation of Docker or Podman" as a prerequisite.

Reply:
yes, the added `* An installation of [command]`podman` or [command]`docker`.` is what was missing.